templates/web/fixmystreet.com/index.html

```diff
@@ -0,0 +1,20 @@
+[% map_js = PROCESS 'front/javascript.html' %]
```

Comment:
My issue was, even if I make fixmystreet.com/footer-marketing.html an empty file (or add some `IF` block to make it empty on the FMS.com homepage at least), there will *still* be an empty `<footer role="contentinfo">` at the bottom of the page ([templates/web/base/footer.html](https://github.com/mysociety/fixmystreet/blob/97a01fcec2c48e379394e0127888edc2e5bbde91/templates/web/base/footer.html#L1-L5)), which seems a shame.

Can you think how we can avoid that?